Sources/Authgear.swift

```diff
@@ -818,10 +818,8 @@ public class Authgear {
     ///   - xState: Use this parameter to provide parameters from the client application to Custom UI. The string in xState can be accessed by the Custom UI. Ignore this parameter if default AuthUI is used
     public func authenticate(
         redirectURI: String,
-        state: String? = nil,
```

Comment:
Apply the same change to `reauthenticate()` `promoteAnonymousUser()`

Reply:
removed `state` from `reauthenticate()` `promoteAnonymousUser()` and added `xState` description to them as well